data/exclusiveCycleLaneStyle.yml

```diff
@@ -0,0 +1,134 @@
+BA: white # https://www.youtube.com/watch?v=NoO8OGArV2s
+
+# BANGLADESH ROAD TRANSPORT AUTHORITY, TRAFFIC SIGNS MANUAL, LANE FOR (cycles and rickshaws) ONLY - Road Marking No F19
+# https://www.rhd.gov.bd/Documents/ConvDocs/Road%20Sign%20Manual%20Volume-1.pdf
+BD: white
+
+# Ministerieel besluit van 11 oktober 1976 houdende de minimum afmetingen en de bijzondere plaatsingsvoorwaarden van de verkeersteken, Hoofdstuk III. Wegmarkeringen, Artikel 16. Overlangse markeringen die een fietspad aanduiden
+# https://www.wegcode.be/nl/regelgeving/1976101105~j6siwtihko#nt4j1l7ivr
+# "De onderbroken streep bestaat uit trekken van ongeveer 0,15 m breedte en van ongeveer 1,25 m lengte, met tussenafstanden van ongeveer 1,25 m."
+BE: white dashes on both sides
+
+# Signalisationsverordnung Art. 74a
+# https://www.fedlex.admin.ch/eli/cc/1979/1961_1961_1961/de#art_74_a
+# (exceptionally also a yellow continuous line, but yellow dashes are the norm)
+CH: yellow dashes
+
+# TP 179 - NAVRHOVÁNÍ KOMUNIKACÍ PRO CYKLISTY, 4 Úseky (intravilán), 4.2.2 Skladebnost - Vyhrazený pruh pro cyklisty
+# https://pjpk.rsd.cz/data/USR_001_2_8_TP/TP_179_2017.pdf
+# see also https://cs.wikipedia.org/wiki/J%C3%ADzdn%C3%AD_pruh_pro_cyklisty
+CZ: white
+
+# Allgemeine Verwaltungsvorschrift zur Straßenverkehrs-Ordnung (VwV-StVO), §10
+# http://www.verwaltungsvorschriften-im-internet.de/bsvwvbund_26012001_S3236420014.htm
+DE: white
+
+DK: white # https://www.google.de/maps/@55.6794357,12.5825915,3a,75y,357.22h,72.01t/data=!3m6!1e1!3m4!1sSm6e_2IJ17W7rSN2AYMeWQ!2e0!7i16384!8i8192
+
+# Liiklusmärkide ja teemärgiste tähendused ning nõuded fooridele, lisa 10
+# https://www.riigiteataja.ee/akt/103032011006 + https://www.riigiteataja.ee/aktilisa/1030/3201/1006/MKM_22022011_M12-lisa10.pdf
+EE: white
+
+# Real Decreto Legislativo 6/2015, de 30 de octubre, por el que se aprueba el texto refundido de la Ley sobre Tráfico, Circulación de Vehículos a Motor y Seguridad Vial, Art 74-79
+# see https://reconstruccionaccidentestrafico.com/vias-ciclistas-definicion-legal-normativa-y-prioridad-de-paso-respecto-a-otros-vehiculos-y-a-peatones/
+ES: white # https://www.google.de/maps/@40.4287197,-3.7009802,3a,75y,271.76h,66.16t/data=!3m6!1e1!3m4!1sRvv1CfHq48gVjwHp-9aw0Q!2e0!7i16384!8i8192
+
+# 729/2018 Tieliikennelaki, Liite 4.1. Pituussuuntaiset tiemerkinnät, K2 Ajokaistaviiva
+# https://finlex.fi/fi/laki/alkup/2018/20180729
+FI: white dashes
+
+FR: white dashes # https://www.google.de/maps/@48.852818,2.3674504,3a,75y,33.46h,64.09t/data=!3m6!1e1!3m4!1sait3Kj3mhuaT0PGthUYGQw!2e0!7i16384!8i8192
+
+GB: white # https://www.google.de/maps/@51.4965114,-0.1255504,3a,75y,346.83h,84.96t/data=!3m6!1e1!3m4!1sty8Zl-R_msKxFjnB702GBw!2e0!7i16384!8i8192
+
+# Zakon o sigurnosti prometa na cestama, Članak 18)
+# https://www.zakon.hr/z/78/Zakon-o-sigurnosti-prometa-na-cestama
+HR: yellow # https://www.google.de/maps/@45.8061213,15.9722997,3a,75y,118.67h,62.19t/data=!3m6!1e1!3m4!1sfJFX3JAwYpYKkjfujlN8jA!2e0!7i13312!8i6656
```

Comment:
```suggestion
# Zakon o sigurnosti prometa na cestama, Članak  2. (1) 18), Članak 112. (1)
# https://narodne-novine.nn.hr/clanci/sluzbeni/2008_06_67_2224.html
# https://narodne-novine.nn.hr/clanci/sluzbeni/2022_07_85_1288.html
# Pravilnik o biciklističkoj infrastrukturi, Članak 40., 48.
# https://narodne-novine.nn.hr/clanci/sluzbeni/2016_03_28_803.html
HR: yellow # https://www.google.de/maps/@45.8090029,15.971595,3a,75y,243.6h,51.8t/data=!3m6!1e1!3m4!1skNmcpBP1iJdD5J_2ZGqEhQ!2e0!7i13312!8i6656
```

Um, I think current file is wrong for `HR`:
- `Članak 18.` (as quoted) seems unrelated, as it talks about semaphores and other lightning, not about cycle lanes? Maybe you though about `Članak  2. (1) 18)` which defines what bicycle lane is, or `Članak 112.(1)` which defines that cyclists must use it, but those only define **_existence_** of exclusive cycle lanes, not their style.

- Firstly, `www.zakon.hr` website is private company (whose business model is combining amendments to a law to a single text, and selling law precedent case decisions) and not official source (despite nice domain name they've grabbed on fist-come-first-served basis!) related to that, law texts on that site are mutable (e.g. what was `Članak 18.` yesterday, might be `Članak 19.` today, or be removed etc.). So best to use official and unmutable source, which is https://narodne-novine.nn.hr/clanci/sluzbeni/xxxx (the disadvantage there however is that sometimes one has to quote several URLs if some law has changed significantly)

- **_style_** of the cycling infrastructure is defined  by sub-law regulation in `.pdf` found at: https://narodne-novine.nn.hr/clanci/sluzbeni/2016_03_28_803.html, e.g. Članak 40., 48. However it only seems to define that the line must be full (not dashed), not it's color (it does say in what cases there must also be red surface). 
 Maybe I'm missing it, or it is defined in some other regulation :man_shrugging: 

  The actual practice on the ground itself is also various (but it does not overly surprise me, as even quite clear regulations about cycling infrastructure are being ignored all the time by the construction contractors without repercussions), but most recognizable would be `red surface+full yellow line` or just `full yellow line` for "cycle lane" on the road ("biciklistička traka" in Croatian):
  - red surface, full yellow line: https://www.google.de/maps/@45.8090029,15.971595,3a,75y,243.6h,51.8t/data=!3m6!1e1!3m4!1skNmcpBP1iJdD5J_2ZGqEhQ!2e0!7i13312!8i6656
  - just a full yellow line: https://www.google.de/maps/@45.8081707,15.972222,3a,75y,17.33h,62.09t/data=!3m6!1e1!3m4!1sTIjQdxBV4OUwZf1Gtw08XQ!2e0!7i13312!8i6656
  or https://www.mapillary.com/app/?lat=45.719591&lng=16.058816&z=17&pKey=1740656346112283&focus=photo
  or https://www.mapillary.com/app/?lat=45.55183606&lng=18.69876068&z=17&pKey=1218993981851809&focus=photo
  - just a full white line: https://www.mapillary.com/app/?lat=45.777645483610996&lng=15.916126705556053&z=17.656886226982298&pKey=949690755844107&focus=photo&x=0.530157050275831&y=0.5816257008

Reply:
Yeah, that's why I wrote "yellow" and not "yellow on red background", because it is not always the case.